src/DevBetterWeb.Web/CustomAttributes/ValidGitHubUrlAttribute.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace DevBetterWeb.Web
+{
+  public class ValidGitHubUrlAttribute : ValidationAttribute
```

Comment:
Follow the Don't Repeat Yourself principle. All but 1 line of each of these attributes is the same. That should have you looking for an abstraction.